src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -118,4 +130,17 @@ public Event createEvent(String eventTitle, String eventLocation, String eventDe
   private EventDateTime createEventDateTime(String time) {
     return new EventDateTime().setDateTime(new DateTime(time)).setTimeZone(TIMEZONE);
   }
+
+  @VisibleForTesting
+  public String getEventEndTime(String startTime, String hours, String minutes) {
```

Comment:
Could you add some more tests for getEventEndTime? There's some logic with computing the end time that you want to make sure doesn't accidentally get broken in the future.